src/components/Error/index.js

```diff
@@ -11,9 +11,9 @@ const Error = ({error}) => {
   const icon = error.type === 'error' ? style.errorIcon : style.warningIcon
   return (
     <div className={classNames(errorStyle, style.container)}>
-      <p className={style.message}>
+      <div>
```

Comment:
Consider: I was probably not clear in my comment, because my point was on having the message inside of a component called `icon`. Happy to chat about it.